test/Jamfile

```diff
@@ -31,6 +34,9 @@ rule do_fail_impl ( a * )
 {
     exit "OpenSSL could not be found. Don't build target fail_if_no_openssl to skip this check" ;
 }
+using openssl : : <search>$(OPENSSL_ROOT)/lib <include>$(OPENSSL_ROOT)/include ;
+alias ssl : : <source>/openssl//ssl/<link>shared ;
+alias crypto : : <source>/openssl//crypto/<link>shared ;
```

Comment:
Why do we need this? There is a `using openssl` before this - we must get rid one of the two.
Are the aliases used anyhow?